services/dyndata.ts

```diff
@@ -6,6 +10,42 @@ const api = $fetch.create({
   baseURL: BASE_URL,
 })
 
+const apiInstanceByPrefix = (prefix: Prefix) => {
+  const endpoint: string = getChainEndpointByPrefix(prefix) || ''
+  return ApiFactory.useApiInstance(endpoint)
+}
+
+const lastIdAssethub = async (prefix: Prefix, collectionId: number) => {
+  const api = await apiInstanceByPrefix(prefix)
+  const ids = await api.query.nfts.item.keys(collectionId)
+  const ints = ids.map(id => Number.parseInt(id.toHuman()?.[1]?.replaceAll(',', '') || '0'))
+  const biggestId = Math.max(...ints)
+  return biggestId
+}
```

Comment:
reuse/combine `getLastIndexUsedOnChain` in `composables/transaction/mintToken/utils.ts`

Reply:
thank you, I use that one instead [3f58f61](https://github.com/kodadot/nft-gallery/pull/11315/commits/3f58f61bbe25bdefa0bcc8bbd8cdd957211ed0e2)